content/configuration/devtool.md

```diff
@@ -49,6 +49,8 @@ W> There are some issues with Source Maps in Chrome. [We need your help!](https:
 
 `cheap-module-source-map` - A SourceMap without column-mappings that simplifies loaded Source Maps to a single mapping per line.
 
+`nosources` - This option prevents the generation of a SourceMap file for your project.
```

Comment:
the description is wrong. it generates a SourceMap but don't include the sources in the SourceMap.